react/src/components/ServiceLauncherModal.tsx

```diff
@@ -184,16 +297,46 @@ const ServiceLauncherModal: React.FC<ServiceLauncherProps> = ({
 
   // Apply any operation after clicking Cancel button
   const handleCancel = () => {
+    form.resetFields();
     onRequestClose();
   };
 
+  const getAIAcceleratorWithStringifiedKey = (resourceSlot: any) => {
+    if (Object.keys(resourceSlot).length <= 0) {
+      return undefined;
+    }
+    const keyName: string = Object.keys(resourceSlot)[0];
+    return {
+      acceleratorType: keyName,
+      // FIXME: temporally convert to number if the typeof accelerator is string
+      accelerator:
+        typeof resourceSlot[keyName] === 'string'
+          ? keyName === 'cuda.shares'
+            ? parseFloat(resourceSlot[keyName])
+            : parseInt(resourceSlot[keyName])
+          : resourceSlot[keyName],
+    };
+  };
+
   return (
     <BAIModal
-      title={t('modelService.StartNewServing')}
-      onOk={handleOk}
-      onCancel={handleCancel}
-      destroyOnClose={true}
+      title={
+        endpoint
+          ? t('modelService.EditModelService')
+          : t('modelService.StartNewServing')
+      }
+      destroyOnClose
       maskClosable={false}
+      footer={() => (
+        <Flex direction="row" justify="end" align="end">
+          <Space size="middle">
```

Comment:
```suggestion
          <Space size="small">
```
The service modal
![image](https://github.com/lablup/backend.ai-webui/assets/28584164/c7132852-362d-4102-a1ea-4c92b18b4851)
Generate token modal
![image](https://github.com/lablup/backend.ai-webui/assets/28584164/822b14ad-e745-4616-8106-b2cbb085dcaa)

If you change the size to small, it looks like below.
![image](https://github.com/lablup/backend.ai-webui/assets/28584164/501810b2-43c4-4a73-b781-a7ba5bad6513)